### PR TITLE
Update Release Notes page with v1.0.0-rc1 info

### PR DESCRIPTION
**Pull request by havan**
Release notes for version `v1.0.0-rc1`.

https://playground.docs.camino.network/release-notes/camino-node/index.html

### CODE REVIEW

File: docs/release-notes/camino-node.md [hunk 1]
```diff
@@ -12,11 +12,51 @@ For more up-to-date information please check [Github Release](https://github.com
 
 :::
 
-## v0.4.9-rc1
+## v1.0.0-rc1
```
Comment (mo-c4t) [inline docs/release-notes/camino-node.md:27]: maybe we need to add (Pre-release)

Comment (havan) [reply docs/release-notes/camino-node.md:27]: Added 👍🏼

File: docs/release-notes/camino-node.md [hunk 2]
```diff
@@ -12,11 +12,52 @@ For more up-to-date information please check [Github Release](https://github.com
...
+
+- Athens Upgrade
+- AddDepositOffer allows creation of deposit offers
+- Whitelist / Owner controlled deposit offers
+- AddressStateTx upgrade to make it MsigAlias capable
+- Fix AddValidatorTx RewardOwner
+- Fix bootstrap of SystemUnlockDepositTx (empty Creds[0])
+- Fix AddddressStateTx Roles
+- Admin API secret (dns attack)
+- Fix bootstrap issues after RewardValTx
```
Comment (mo-c4t) [inline docs/release-notes/camino-node.md]: already in v0.4.9

File: docs/release-notes/camino-node.md [hunk 3]
```diff
@@ -12,11 +12,52 @@ For more up-to-date information please check [Github Release](https://github.com
...
+- Change Deposit API (query with timestamp) and strings for uint64 result types.
+
```
Comment (mo-c4t) [inline docs/release-notes/camino-node.md:47]: Fix Validator reconnection when public IP changes

Comment (havan) [reply docs/release-notes/camino-node.md:47]: Added 👍🏼

File: docs/release-notes/camino-node.md (hunk 2 above)
Comment (havan) [reply docs/release-notes/camino-node.md]: Fixed

**Pull request merged**